mula/scheduler/context/context.py

```diff
@@ -42,20 +43,20 @@ def __init__(self) -> None:
 
         # Services
         katalogus_service = services.Katalogus(
-            host=self._remove_trailing_slash(str(self.config.host_katalogus)),
+            host=remove_trailing_slash(str(self.config.host_katalogus)),
```

Comment:
I recommend considering the new Pydantic v2 branch for my proposed solution. In this implementation, clients take on the responsibility of handling base URL manipulation by stripping or preparing it. This approach reduces verbosity, as the callee is relieved of the task of dealing with these details. The expectation is that clients will automatically append endpoints to base URLs, assuming they lack a trailing slash. This simplifies the caller's code, making it more intuitive and concise.

Reply:
Would it be advisable to implement that change during this pr or after it has been merged?

Reply:
It doesn't matter too much, but I think it's preferable to align as soon as possible to keep the consistency